src/library-authoring/generic/manage-collections/messages.tsx

```diff
@@ -0,0 +1,51 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
```

Comment:
Nit: this file can be `.ts` not `.tsx`.

Reply:
Fixed: 0962eabcf8fbbbb841cd0db3bb179f954ab8ea4a